CMakeLists.txt

```diff
@@ -189,7 +189,7 @@ if("${gRPC_PROTOBUF_PROVIDER}" STREQUAL "module")
       set(_gRPC_PROTOBUF_PROTOC_LIBRARIES libprotoc)
     endif()
     if(TARGET protoc)
-      set(_gRPC_PROTOBUF_PROTOC protoc)
+      set(_gRPC_PROTOBUF_PROTOC ${PROTOBUF_ROOT_DIR}/protoc)
```

Comment:
This variable is set to:
    set(PROTOBUF_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/third_party/protobuf)
and that directory gets protoc placed in it:
$ ls third_party/protobuf/protoc -l
-rwxr-xr-x 1 pi pi 9397460 Jul 20 18:53 third_party/protobuf/protoc

At least this is what happens when I build with this setting